packages/grid/x-data-grid/src/components/containers/GridToolbarContainer.tsx

```diff
@@ -27,6 +27,7 @@ const GridToolbarContainerRoot = styled('div', {
 })(({ theme }) => ({
   display: 'flex',
   alignItems: 'center',
+  flexWrap: 'wrap',
   padding: theme.spacing(0.5, 0.5, 0),
```

Comment:
The input bottom border touches the column headers if we leave as `0px`.

![image](https://user-images.githubusercontent.com/42154031/167148631-da4a590d-a459-4a92-b177-b9328a488b74.png)

```suggestion
  padding: theme.spacing(0.5),
```

Reply:
If I do that, it will also modify the Toolbar even when there is no quick filter

What about adding `paddingBottom:theme.spacing(0.5)` to the quick filter component?

Reply:
IMO the bottom padding should be there since the beginning. The already existent buttons also touch the border.

<img width="400" src="https://user-images.githubusercontent.com/42154031/167170826-8c0593ff-c3ea-4536-bdb3-ab4908e8608a.png" alt="image" style="max-width: 100%;">